frappe/public/js/frappe/ui/field_group.js

```diff
@@ -23,6 +23,23 @@ frappe.ui.FieldGroup = class FieldGroup extends frappe.ui.form.Layout {
 			this.refresh();
 			// set default
 			$.each(this.fields_list, function(i, field) {
+				if (field.df["fieldtype"] == 'Date') {
+					// set min date
+					if (field.df["minDate"]) {
+						console.log('minDate');
+						field.datepicker.update({
+							minDate: new Date(field.df["minDate"])
+						});
+					}
+					// set max date
+					if (field.df["maxDate"]) {
+						console.log('maxDate');
+						field.datepicker.update({
+							maxDate: new Date(field.df["maxDate"])
+						});
+					}
+				}
```

Comment:
Move this to date.js instead?

I've already done it here
https://github.com/frappe/frappe/pull/17021/files#diff-0f4239a698d6aac7590fd053db65d9164d6934b803a9a07983fcc56e2a640a2bR67